source/index.html.md

```diff
@@ -940,6 +1018,34 @@ To determine if a line item was matched, use the productCodeMatched attribute:
 | files.apInvoice.departmentCode            |  The department code of this invoice.             |
 | files.apInvoice.branchCountry            |   The branch country of this invoice.             |
 
+### *Files/apInvoice/Cluster* attributes
+
+| Attribute                               |  Description                                                                                                                      |
+| --------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| files.apInvoice.cluster.total   | The cluster total Value                                                      |
```

Comment:
Each sub-total section on the invoice is considered a cluster, So cluster total basically is a sub-total